test/base-tests/upload-to-aws.sh

```diff
@@ -0,0 +1,151 @@
+#!/bin/bash
+set -xeuo pipefail
+
+source /etc/os-release
+
+# Fedora 31's composer-cli doesn't support doing image uploads.
+if [[ $ID == fedora ]] && [[ $VERSION_ID == 31 ]]; then
+    echo "Fedora 31 does not support image uploads with composer-cli."
+    exit 0
+fi
+
+# Colorful output.
+function greenprint {
+    echo -e "\033[1;32m${1}\033[0m"
+}
+
+# Apply lorax patch to work around pytoml issues in RHEL 8.x.
+# See BZ 1843704 or https://github.com/weldr/lorax/pull/1030 for more details.
+if [[ $ID == rhel ]]; then
+    sudo sed -r -i 's#toml.load\(args\[3\]\)#toml.load(open(args[3]))#' \
+        /usr/lib/python3.6/site-packages/composer/cli/compose.py
+    sudo rm -f /usr/lib/python3.6/site-packages/composer/cli/compose.pyc
+fi
+
+# We need jq for parsing composer-cli output.
+if ! hash jq; then
+    greenprint "Installing jq"
+    sudo dnf -qy install jq
```

Comment:
Why `-q` here?

Reply:
Also, why single out jq here? We have lots of dependencies, but we just check for jq? Can't we rely on the target image to contain it already?

Reply:
I don't know, the definition of the images we use is somewhere in osbuild organization, but I lost track on where it lives. Maybe @major knows from top of his head?